.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## What
+Write a summary of the changes made in the PR.
+
+## Why
+Exlpain the reasons for the changes.
+
+## Type of change
+Select multiple if applicable.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause a breaking change)
+- [ ] Tests (adds or updates tests)
+- [ ] Documentation (adds or updates documentation)
+- [ ] Refactor (style improvements, performance improvements, code refactoring)
+- [ ] Revert (reverts a commit)
+- [ ] CI/Build (adds or updates a script, change in external dependencies)
+
+## Dependency Change
+If a new dependency is being added, please ensure that it adheres to the following guideline https://github.com/apimatic/apimatic-codegen/wiki/Policy-of-adding-new-dependencies-in-the-core-libraries
+
+## Breaking change
+If the PR is introducing a breaking change, please ensure that it adhares to the following guideline 
```

Comment:
Please add the following guideline at the end of this line https://github.com/apimatic/apimatic-codegen/wiki/Guidelines-for-maintaining-core-libraries